Open Judge System/Web/OJS.Web/Controllers/TempController.cs

```diff
@@ -374,6 +374,9 @@ public async Task<ActionResult> MigrateContestCategoryFromRemoteJudge(string id)
 
         public ActionResult UpdateParticipantsTotalScore()
         {
+            // This request causes 504 (Timeout error)
+            return this.Content("UPDATED");
```

Comment:
"UPDATED" is not correct. "Unsupported" seems more legit. Also add comment in the code that the query will work if executed in the database directly.